bank2ynab/ynab_api.py

```diff
@@ -1,29 +1,38 @@
-import requests
 import json
-import configparser
 import logging
+from configparser import DuplicateSectionError, NoSectionError
+
+import requests
 
-import b2y_utilities
+from config_handler import ConfigHandler
 
 # configure our logger
 logging.basicConfig(format="%(levelname)s: %(message)s", level=logging.INFO)
 
 
-class YNAB_API(object):  # in progress (2)
+class YNAB_API:
+    # TODO - revise docstring
```

Comment:
This class is too unwieldy and needs to be broken up - maybe handle the API responses elsewhere?